include/api.php

```diff
@@ -28,11 +29,8 @@
  * @return null|string|array null if displaying, array if raw is requested, string otherwise
  */
 function pll_the_languages( $args = '' ) {
-	if ( PLL() instanceof PLL_Frontend ) {
-		$switcher = new PLL_Switcher();
-		return $switcher->the_languages( PLL()->links, $args );
-	}
-	return '';
```

Comment:
Revert this code because of the direct use of PLL_Switcher::the_languages method to render the language switcher block.

I don't believe a API `pll_switcher block` function is needed because `pll_the_languages` function already does the job.